docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+services:
+  node: 
+    image: victorabz/the-spinners
+    build: .
+    ports: 
+      - 80:3000
+    command: bash -c 'while !</dev/tcp/db/5432; do sleep 1; done; npm start'
```

Comment:
What does this command do?

Reply:
the server is started with `npm start` once the database is up and properly running (while file not exists, sleep

Reply:
The `depends_on` property ensures that the listed services are properly started before the parent service is started. You can remove line 8 instead